scripts/skills/perks/perk_rf_offhand_training.nut

```diff
@@ -104,10 +104,7 @@ this.perk_rf_offhand_training <- ::inherit("scripts/skills/skill", {
 
 	function onPayForItemAction(_skill, _items)
 	{
-		if (_skill != null && _skill.getID() != "perk.rf_target_practice")
-		{
-			this.m.IsSpent = true;
-		}
+		this.m.IsSpent = true;
```

Comment:
This has nothing to do with this PR and should be moved to another PR

Reply:
This perk used to be able to allow swapping ammo for free. This was then accounted for in `rf_offhand_training` where if you are swapping ammo with this perk then that perk didn't become "spent". Now that we removed this ability from the target practice perk, we have to remove that accounting for it in the other perk too. So it is part of this change.